templates/include/telemetry/syncItem.1.schema.json

```diff
@@ -0,0 +1,106 @@
+{
+  "type": "object",
+  "required": ["when", "took"],
+  "properties": {
+    "didLogin": { "type": "boolean" },
+    "when": { "type": "integer" },
+    "devices": {
+      "type": "array",
+      "items": { 
+        "required": ["os", "id", "version"],
+        "additionalProperties": false,
+        "type": "object",
+        "properties": {
+          "id": { "type": "string", "pattern": "^[0-9a-f]{64}$" },
+          "os": { "type": "string" },
+          "version": { "type": "string" }
+        }
+      }
+    },
+    "status": {
+      "type": "object",
+      "anyOf": [
+        { "required": ["sync"] },
+        { "required": ["service"] }
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "sync": { "type": "string" },
+        "service": { "type": "string" }
+      }
+    },
+    "why": { "type": "string" },
+    "took": { "type": "integer", "minimum": -1 },
+    "failureReason": @TELEMETRY_SYNCERROR_1_JSON@ ,
+    "engines": {
+      "type": "array",
+      "minItems": 0,
+      "items": { 
+        "type": "object",
+        "required": ["name"],
+        "additionalProperties": false,
+        "properties": {
+          "failureReason": @TELEMETRY_SYNCERROR_1_JSON@ ,
+          "name": { "type": "string" },
+          "took": { "type": "integer", "minimum": 1 },
```

Comment:
Is this the correct minimum value? It seems like this is causing the majority of the validation errors:

```json
{
        "telemetry.sync.4": {
            "error_count": 57,
            "error_rate": 5.7,
            "errors": {
                "Validation Error: ('additionalProperties', '#/properties/payload/properties/syncs/items/properties/engines/items', '#/payload/syncs/0/engines/2/steps')": 1,
                "Validation Error: ('additionalProperties', '#/properties/payload/properties/syncs/items/properties/engines/items', '#/payload/syncs/0/engines/3/steps')": 1,
                "Validation Error: ('additionalProperties', '#/properties/payload/properties/syncs/items/properties/engines/items', '#/payload/syncs/0/engines/4/steps')": 1,
                "Validation Error: ('minimum', '#/properties/payload/properties/syncs/items/properties/engines/items/properties/took', '#/payload/syncs/0/engines/1/took')": 2,
                "Validation Error: ('minimum', '#/properties/payload/properties/syncs/items/properties/engines/items/properties/took', '#/payload/syncs/0/engines/2/took')": 48,
                "Validation Error: ('minimum', '#/properties/payload/properties/syncs/items/properties/engines/items/properties/took', '#/payload/syncs/0/engines/3/took')": 1,
                "Validation Error: ('required', '#/properties/payload/properties/syncs/items', '#/payload/syncs/0')": 1,
                "Validation Error: ('required', '#/properties/payload/properties/syncs/items/properties/devices/items', '#/payload/syncs/0/devices/0')": 2
            },
            "time": 0.64,
            "total": 1000
        }
}